meinberlin/apps/budgeting/assets/helpers.js

```diff
@@ -0,0 +1,20 @@
+import django from 'django'
+
+const months = [
+  django.gettext('January'),
+  django.gettext('February'),
+  django.gettext('March'),
+  django.gettext('April'),
+  django.gettext('May'),
+  django.gettext('June'),
+  django.gettext('July'),
+  django.gettext('August'),
+  django.gettext('September'),
+  django.gettext('October'),
+  django.gettext('November'),
+  django.gettext('December')
+]
+export const toDate = (isodate) => {
```

Comment:
What is the isodate you use? It should have a timezone/date offset. We should be very careful with that in our js.

Reply:
I think we should use get_format? https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#internationalization-in-javascript-code

Reply:
we receive a string that looks like this `YYYY-MM-DDTHH:mm:ss.sssZ`, which i think is the ISO notation?
https://262.ecma-international.org/5.1/#sec-15.9.1.15

do you know by heart or have some information to the timezone offset, which is problematic inJS?

Anyways I will try out the get_format from Django.

Reply:
Ah, then the timezone-stuff is already taken over by the serializer. :) 
But the get_format should still help to show the date in an internationalized format. (and we should also check that out for the comments...)

Reply:
@khamui @fuzzylogic2000 should this befor we merge?